gtd-timer/Middleware/ErrorHandlingMiddleware.cs

```diff
@@ -55,6 +55,9 @@ private HttpStatusCode GetExceptionCode(Exception exception)
                 case UserAlreadyExistsException _:
                     exceptionCode = HttpStatusCode.Conflict;
                     break;
+                case IncorectLoginException _:
```

Comment:
this can be combined with IncorrectPasswordException case - they do the same thing

Reply:
now if I think about it - IncorrectPasswordException  sounds suspicious. maybe it should be removed now

Reply:
the problem is that we indid need that exeption because @nataliapachva is curently creating a setting page where we need to write an old password to change password